pandas/tests/test_generic.py

```diff
@@ -1851,6 +1851,15 @@ def test_pipe_panel(self):
         with tm.assertRaises(ValueError):
             result = wp.pipe((f, 'y'), x=1, y=1)
 
+    def test_pct_change(self):
+        pnl = DataFrame([np.arange(0, 40, 10), np.arange(0, 40, 10), np.arange(0, 40, 10)]).astype(np.float64)
+        pnl.iat[1,0] = np.nan
+        pnl.iat[1,1] = np.nan
+        
+        expected = pnl.ffill(axis=1).pct_change(axis=1, fill_method=None)
```

Comment:
test with axis 0, 2 as well (if their are existing tests for that, pls move them here)

further this should be a bit more generic (or append _frame) to the test name as the ``generic.py`` tests should test Series,DataFrame,Panel